index.d.ts

```diff
@@ -13,7 +13,7 @@ declare namespace callsites {
 		/**
 		Returns the current function.
 		*/
-		getFunction(): Function | undefined;
+		getFunction(): () => void;
```

Comment:
I don't think this is correct.

Reply:
what is? I'm no ts expert, it was complaining about it...

Reply:
Two things. You're removing the optionality. And you cannot know that the function returns void. You need an AnyFunction.

Reply:
how do i write a AnyFunction? all i get is:

```

  index.d.ts:16:18
  ✖  16:18  Don't use Function as a type. Use a specific function type instead, like () => void.  @typescript-eslint/ban-types

  index.test-d.ts:9:13
  ✖   9:13  Don't use Function as a type. Use a specific function type instead, like () => void.  @typescript-eslint/ban-types

```